src/core/propertyTypes.js

```diff
@@ -127,3 +131,24 @@ function srcParse (value) {
 function vecParse (value) {
   return coordinates.parse(value, this.default);
 }
+
+function updateCamera(time, component) {
+  var camera = component.el.sceneEl.camera.el.getObject3D('camera');
+  var position = camera.position.clone();
+  position.setFromMatrixPosition(camera.matrixWorld); 
+  return position;
+}
+
+function updateTime(time) {
+  return time;
+}
+
+function updateResolution(time, component) {
```

Comment:
seems expensive to do on each tick, seems like this would update only on window resize. shadertoy is able to change resolution, but don't know if that's needed in aframe